tidysic/settings/formatted_string.py

```diff
@@ -1,55 +1,106 @@
 import re
+from abc import ABC, abstractmethod
 
 from tidysic.file.taggable import Taggable
 
 
+class _Unit(ABC):
+
+    @abstractmethod
+    def write(self, taggable: Taggable):
+        pass
+
+
+class _SubstitutableUnit(_Unit):
```

Comment:
For my own curiosity: why using "private" classes? Is there such a thing in Python?

Reply:
I simply did it to express that these classes aren't intended to be imported from this file. Their purpose is to be used in this file and this file only. I could have them be declared inside the `FormattedString` class, but decided against it in order to keep their names short.